client/pages/sections/protocols/sections.js

```diff
@@ -117,16 +117,20 @@ const ProtocolSections = ({ sections, protocolState, editable, newComments, ...p
   return (
   <Accordion open={getOpenSection(protocolState, editable, sections)} toggleAll={!props.pdf}>
     {
-      Object.keys(sections).sort(sortGranted(sections, props.isGranted)).filter(section => !sections[section].show || sections[section].show(props)).map((section, sectionIndex) => (
-        <Fragment key={section}>
-          <ChangedBadge fields={sections[section].fields.map(field => field.name)} />
-          <ExpandingPanel alwaysUpdate={section === 'conditions' || section === 'authorisations'} key={section} title={getTitle(sections[section], newComments, props.values)}>
-            {
-              getSection(section, { ...props, protocolState, editable, ...sections[section], sectionsLength: size(sections), sectionIndex })
-            }
-          </ExpandingPanel>
-        </Fragment>
-      ))
+      Object.keys(sections)
+        .sort(sortGranted(sections, props.isGranted))
+        .filter(section => !sections[section].show || sections[section].show(props))
+        .map((section, sectionIndex) => {
+          const fields = (sections[section].fields || []).map(field => field.name);
```

Comment:
This is the only substantive change here - adding ` || []` before the map.

The rest of it is adding whitespace so I could make any sense of the code.